packages/app/src/messages/selectors/get-thread-draft-messages.selector.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { createSelector } from "reselect"
+import { MessageType } from "App/messages/constants"
+import { Message } from "App/messages/dto"
+import { messagesStateSelector } from "App/messages/selectors/messages-state.selector"
+
+export const getThreadDraftMessagesSelector = (threadId: string) =>
+  createSelector(
+    messagesStateSelector,
+    ({ messageIdsInThreadMap, messageMap }) => {
+      const messageIds = messageIdsInThreadMap[threadId] ?? []
+      return messageIds
+        .map((messageId) => messageMap[messageId])
+        .find((message: Message) => message.messageType === MessageType.DRAFT)
```

Comment:
Not 100% sure if I get it right, but the test description says that `returns last draft message if many exists`, while `find` returns the value of the first element that passes a test. Is it ok?

Reply:
https://github.com/mudita/mudita-center/pull/835/commits/31b4aae7dbdc63e21c41fb1e179a2ea58b9d9d98